conda/environments/cudf_dev_cuda9.2.yml

```diff
@@ -6,9 +6,9 @@ channels:
 - conda-forge
 - defaults
 dependencies:
-- nvstrings=0.9.*
```

Comment:
still want these as `nvstrings` should continue to be a separate package for the time being